benchmarks/kotlin-ktor/README.md

```diff
@@ -0,0 +1,104 @@
+# Kotlin-Ktor benchmark
+
+The benchmark is designed to simulate a chat application where multiple clients send requests simultaneously.
+It uses the [Ktor](https://ktor.io/) framework.
+
+Namely, the benchmark runs multiple clients at the same time.
+Clients are set up with one or both of the following tasks:
+
+- Join a chat and send a randomized message to it.
+- Create a private chat with another user (if not already created) and send a randomized message to them.
+
+They repeat those task the required number of times.
+After that number of successfully completed tasks is validated.
+
+It's also possible to implement validation via end-to-end comparison of the artifacts is produced.
+However, in our case it would be tricky to implement, since every run is randomized both in content and in the order of
+operations, so a semantic comparison of logs would be required.
+Gladly, it's not needed for us, since each task auto-validates itself, so it will successfully terminate only if
+everything went as expected.
+Hence, counting the number of successfully completed tasks is enough to validate the run.
+
+## Running the benchmark
+
+To run benchmark, execute the following command:
+
+```bash
+java -jar <renaissance.jar> kotlin-ktor
+```
+
+Here are the arguments that can be passed to the benchmark:
+
+- `client_count`: Number of clients that are simultaneously sending the requests.
+  Increasing number of clients (with proportional decrease in number of repetitions) doesn't have much effect on the
+  overall runtime. Default value is the number of available CPUs.
+- `iterations_count`: Number of times clients should repeat their designated operations. Default value is 2000.
```

Comment:
One operation is always one message, right? If that's the case, maybe we should name this parameter `messages_count` to avoid any confusion with benchmark iterations.

Reply:
No, one operation could be represented by multiple messages.
For example, to send a private message to another user, you may first need to create a private chat with them.
To create a chat, a `CreateDirectMessageChatCommand` must be sent first.

I'll rephrase from "operations" to "tasks"